cmake/platform.cmake

```diff
@@ -137,7 +137,14 @@ elseif(UNIX OR MINGW)
             append(CMAKE_CCXX_SANITIZER_FLAGS "-g -fno-omit-frame-pointer")
         endif()
     elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-        set(DEF_ARCH_OPT_FLAGS "-msse4.1")
+        if(TARGET_ARCH STREQUAL "AARCH64")
+             set(DEF_ARCH_OPT_FLAGS "-O3 -mcpu=native")
+             set(DNNL_ENABLE_JIT_PROFILING CACHE BOOL "OFF" FORCE)
+             set(DNNL_ENABLE_JIT_DUMP CACHE BOOL "OFF" FORCE)
```

Comment:
`DNNL_ENABLE_JIT_DUMP` is not a cmake option, so this line effectively does nothing.
I might miss something, but it seems there is not reason to change anything wrt `DNNL_ENABLE_JIT_DUMP` in this PR.

Reply:
You're right. Fixed.